_data/engine-cli/docker_pull.yaml

```diff
@@ -181,7 +181,7 @@ examples: |-
 
   ```dockerfile
   FROM ubuntu@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2
-  MAINTAINER some maintainer <maintainer@example.com>
+  LABEL maintainer="some maintainer <maintainer@example.com>"
```

Comment:
This YAML file is generated from sources in the docker/cli repository, so would have to be made there; are you interested in opening a pull request there? https://github.com/docker/cli/blob/master/docs/reference/commandline/pull.md

Note that the CLI repository requires commit messages to have a `Signed-off-by` line; see instructions here; https://github.com/docker/cli/blob/master/CONTRIBUTING.md#sign-your-work

Also happy to make a PR for you if you prefer

Reply:
Opened a PR there. Thank you!